playground/infrastructure/cloudbuild/playground_cd_examples.sh

```diff
@@ -97,15 +97,15 @@ LogOutput "Installing python and dependencies."
 export DEBIAN_FRONTEND=noninteractive
 apt install -y apt-transport-https ca-certificates software-properties-common curl unzip apt-utils > /dev/null 2>&1
 add-apt-repository -y ppa:deadsnakes/ppa > /dev/null 2>&1 && apt update > /dev/null 2>&1
-apt install -y python3.8 python3.8-distutils python3-pip > /dev/null 2>&1
-apt install -y --reinstall python3.8-distutils > /dev/null 2>&1
+apt install -y python3.9 python3.9-distutils python3-pip > /dev/null 2>&1
```

Comment:
FYI, I don't know if python3.9-distutils is available (https://github.com/deadsnakes/issues/issues/150), and not sure if this package is still necessary.

Reply:
is this code exercised by some IT suite?

Reply:
I believe these scripts are invoked for playground deployment, not exercised by a test. Looks like we can get away with python3-distutils